Layout update

bvreede opened the pull request:
This pull request updates:
- The layout of the cards on the main page
- Explanatory text for the data and three data categories on the main page
- Text style is standardized with `prose`
- The grid style of the Topical page cards is updated
- A footer text is added to the main page

Peter9192 commented on nuxt.config.js:
```diff
@@ -67,7 +67,26 @@ export default {
   tailwindcss: {
     config: {
-      plugins: [tailwindTypography]
+      plugins: [tailwindTypography],
+      theme: {
+        typography: {
+          DEFAULT: { // edit the prose class
+            css: {
+              'font-weight': '100',
+              p: {
+                'font-weight': '100',
+                color: '#2b32a4'
+              },
+              a: {
+                'font-weight': '300',
+                '&:hover': {
+                  color: '#a0a0a0'
+                }
+              }
+            }
+          }
+        }
+      }
```
Does this overwrite the defaults or replace them? I would think we want to overwrite only the elements that we want to be different, and not "clear" all the other defaults.

Peter9192 reviewed (changes requested):
Really nice and shiny new landing page! Can you still fix the prose class for the data cards under "topical"?

Peter9192 commented on nuxt.config.js:
```diff
@@ -67,7 +67,26 @@ export default {
-      plugins: [tailwindTypography]
+      plugins: [tailwindTypography],
+      theme: {
+        typography: {
+          DEFAULT: { // edit the prose class
```
could it be that you need to insert an "extend" keyword here? See https://tailwindcss.com/docs/typography-plugin#customizing-the-css

bvreede replied:
That was it! Thanks :)

bvreede commented on pages/index.vue:
```diff
@@ -1,41 +1,50 @@
         </NuxtLink>
-        <p class="p-4">
-          Convection-permitting climate model output
+        <p class="prose p-4">
```
Descendants inherit the class, but the parent div also contains the headers of the cards, and they do not look good in prose.

bvreede commented:
Thanks for the feedback @Peter9192! I have made the requested changes, as well as an additional tweak to the image cards: the entire card is now clickable, and I have added a grayscale hover to the card.

Because the entire card is now one link, I have removed the grayscale hover on the header text, and the small loss is that the hover color of the card header background is less nice (used to be light gray, now it's very dark, almost black). I like it a bit less, but having different things happen on different hover locations is confusing, so I left it at this.

Peter9192 reviewed (approved):
This looks much better. I noticed two more things, could you pick up on those in follow-up PRs?

- The (home) page now scrolls instead of scaling down. I would like it if at least the main page doesn't need scrollbars
- The topical cards are spaced very far apart on a (very) wide screen. A different alignment option might be nicer.

Peter9192 commented on pages/index.vue:
```diff
@@ -1,41 +1,50 @@
-  <div class="flex flex-col place-content-evenly h-full p-24">
-    <div class="flex gap-48 place-content-center h-full">
-      <div class="border-4 rounded-lg w-1/3 min-h-2/3 p-8">
+  <div class="flex flex-col p-24">
```
Does reinstating the h-full here fix the scrollbar problem?

Peter9192 replied:
Oh I think it's the `w-full` on the images.

bvreede replied:
Can we look at this in ± an hour with screenshare? I don't think I fully understand the issue so I'm not sure how to fix.

bvreede commented:
> This looks much better. I noticed two more things, could you pick up on those in follow-up PRs?
> 
> * The (home) page now scrolls instead of scaling down. I would like it if at least the main page doesn't need scrollbars
> * The topical cards are spaced very far apart on a (very) wide screen. A different alignment option might be nicer.

Made #63 for the second issue, thanks! Good point, I'll see if I can address this properly when I'm working on a larger screen :)
